lib/subwayside_ui_web/controllers/trains_live.ex

```diff
@@ -199,35 +233,79 @@ defmodule SubwaysideUiWeb.TrainsLive do
-        passengers <= SubwaysideUi.Car.aw2(car) -> {2, "Some crowding"}
-        true -> {3, "Crowded"}
+    gtfs_crowding_five_category =
+      if Map.has_key?(assigns.gtfs_crowding, car.car_nbr) do
+        assigns.gtfs_crowding[car.car_nbr].five_level_crowding
+      else
+        0
+      end
+
+    gtfs_crowding_three_category =
+      if Map.has_key?(assigns.gtfs_crowding, car.car_nbr) do
+        assigns.gtfs_crowding[car.car_nbr].three_level_crowding
+      else
+        0
+      end
+
+    occupancy_percentage =
+      if Map.has_key?(assigns.gtfs_crowding, car.car_nbr) do
+        assigns.gtfs_crowding[car.car_nbr].occupancy_percentage
+      else
+        0
       end
 
-    filled_classes = for _ <- 1..filled_pins, do: "hero-user-solid"
-    unfilled_classes = for _ <- 2..filled_pins//-1, do: "hero-user"
+    gtfs_three_crowding_text = get_gtfs_three_crowding_text(gtfs_crowding_three_category)
+
+    gtfs_five_crowding_text = get_gtfs_five_crowding_text(gtfs_crowding_five_category)
+
+    gtfs_three_filled_classes = List.duplicate("hero-user-solid", gtfs_crowding_three_category)
+
+    unfilled_three =
+      if length(gtfs_three_filled_classes) != 0,
+        do: 3 - length(gtfs_three_filled_classes),
+        else: 0
+
+    gtfs_three_unfilled_classes = List.duplicate("hero-user", unfilled_three)
+
+    gtfs_five_filled_classes = List.duplicate("hero-user-solid", gtfs_crowding_five_category)
+
+    unfilled_five =
+      if length(gtfs_five_filled_classes) != 0, do: 5 - length(gtfs_five_filled_classes), else: 0
+
+    gtfs_five_unfilled_classes = List.duplicate("hero-user", unfilled_five)
 
     assigns =
       assigns
       |> assign(:weight, car.weight)
       |> assign(:empty_weight, car_base_weight)
       |> assign(:seated_capacity, seated_capacity)
       |> assign(:passengers, passengers)
-      |> assign(:pin_classes, filled_classes ++ unfilled_classes)
-      |> assign(:crowding, crowding)
+      |> assign(:occupancy_percentage, occupancy_percentage)
+      |> assign(:gtfs_three_crowding_text, gtfs_three_crowding_text)
+      |> assign(:gtfs_five_crowding_text, gtfs_five_crowding_text)
+      |> assign(:gtfs_five_pin_classes, gtfs_five_filled_classes ++ gtfs_five_unfilled_classes)
+      |> assign(:gtfs_three_pin_classes, gtfs_three_filled_classes ++ gtfs_three_unfilled_classes)
 
     ~H"""
     <tr>
       <td class="pl-2 pr-2"><%= @car.car_nbr %></td>
       <td class="pl-2 pr-2"><%= @passengers %></td>
       <td class="pl-2 pr-2">
-        <span class="inline">
-          <span :for={class <- @pin_classes} class={"#{class} inline-block w-5"} />
+        <span title={@gtfs_three_crowding_text} class="inline">
+          <span :for={class <- @gtfs_three_pin_classes} class={"#{class} inline-block w-5"} />
+        </span>
+        <br />
+        <%!-- <%= @gtfs_three_crowding_text %> --%>
+      </td>
+      <td class="pl-2 pr-2">
+        <span title={@gtfs_five_crowding_text} class="inline">
```

Comment:
I thought the tooltip was cleaner than also including the text, but I'm not sure if we should do anything else here w/ accessibility in mind.